userspace/falco/app_actions/init_outputs.cpp

```diff
@@ -23,6 +23,11 @@ using namespace falco::app;
 
 application::run_result application::init_outputs()
 {
+	if (m_state->config->m_outputs.empty())
+	{
+		return run_result::fatal("No outputs configured. Please configure at least one output file output enabled but no filename in configuration block");
```

Comment:
Mmmh is this sentence ok? :laughing:

Reply:
This was moved as-is from the configuration parsing code. You're right, let me rephrase this better 😂